examples/backwards_compatibility/BUILD

```diff
@@ -1,19 +0,0 @@
-package(default_visibility = ["//tests:__subpackages__"])
```

Comment:
hm. are we sure we want to remove backwards compat testing? this effectively means we can unknowingly break backwards compatibility with old @dataform/core versions.

Reply:
+1 we should keep this

Reply:
I'd say remove the tests when we remove the support, but AIUI we still have it, and when we do remove it we should do a major version bump

Reply:
SGTM